service-mesh/control-plane/base/init-job.yaml

```
@@ -0,0 +1,92 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: odh-ossm-init-job
+spec:
+  template:
+    spec:
+      containers:
+      - name: odh-ossm-init-job
+        image: quay.io/openshift/origin-cli
+        command: ["/bin/bash", "-c", "--"]
+        args:
+          - |
+            mkdir /tmp/bin
+
+            # install jq for json parsing. 
+            wget https://stedolan.github.io/jq/download/linux64/jq  -O /tmp/bin/jq  && chmod +x /tmp/bin/jq 
+
+            # install envsubst 
+            curl -L https://github.com/a8m/envsubst/releases/download/v1.2.0/envsubst-`uname -s`-`uname -m` -o /tmp/bin/envsubst && chmod +x /tmp/bin/envsubst
+            export PATH=/tmp/bin:$PATH
+
+            export CLIENTS_SECRET=$(openssl rand -hex 32)
+            export HMACS_SECRET=$(openssl rand -hex 32)
```

```suggestion
            export CLIENT_SECRET=$(openssl rand -hex 32)
            export HMAC_SECRET=$(openssl rand -hex 32)
```

See the comment in `oauth-configmap.yaml`